ur3e_servo:
  ros__parameters:
    robot_description_kinematics:
      ur_manipulator:
        kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
        kinematics_solver_search_resolution: 0.005
